docs/docsite/rst/dev_guide/collections_tech_preview.rst

```diff
@@ -271,20 +271,20 @@ will be the version displayed everywhere in Galaxy; however, users will still be
 Installing collections
 ----------------------
 
-You can use the ``ansible-galaxy collection install`` command to install a collection on your system. The collection by default is installed at ``/path/ansible_collections/my_namespace/my_collection``. You can optionally add the ``-p`` option to specify an alternate location.
+You can use the ``ansible-galaxy collection install`` command to install a collection on your system. You must specify an installation location using the ``-p`` option.
 
 To install a collection hosted in Galaxy:
 
 .. code-block:: bash
 
-   ansible-galaxy collection install my_namespace.my_collection -p /path
+   ansible-galaxy collection install my_namespace.my_collection -p /collections
 
 
 You can also directly use the tarball from your build:
 
 .. code-block:: bash
 
-   ansible-galaxy collection install my_namespace-my_collection-1.0.0.tar.gz -p ./collections/ansible_collections
```

Comment:
I had this example to show that you could specify `ansible_collections` in the path and it would be installed there instead of `ansible-galaxy` appending the path.

Reply:
@jborean93 but wouldn't that be equivalent? If we have to use the magic path with `ansible_collections` in it, I'd rather not show people something that could be more confusing (e.g. why would I put it in the path sometimes but not others—like do I have to explicitly type this in if installing from a tarball vs installing from galaxy?)

Or are you saying you _have_ to put the full path if installing from a tarball?

Reply:
Happy to leave that out as the note below documents that anyway.